packages/examples/packages/send-flow/src/index.tsx

```diff
@@ -98,7 +98,29 @@ export const onUserInput: OnUserInputHandler = async ({
   if (event.type === UserInputEventType.InputChangeEvent) {
     switch (event.name) {
       case 'amount':
-      case 'to':
+      case 'to': {
+        // For testing purposes, we display the avatar of the zero address.
+        if (event.value === '0x0000000000000000000000000000000000000000') {
```

Comment:
Could we just check that the value matches `HexChecksumAddressStruct` and use that for `displayAvatar`?

Reply:
Done.